bin/setup_dot1x

```diff
@@ -3,14 +3,14 @@
 ROOT=$(dirname $0)/..
 cd $ROOT
 
-sudo rm -rf inst/faucet/faucet_dot1x
-mkdir -p inst/faucet/faucet_dot1x
+#sudo rm -rf inst/faucet/daq-faucet-dot1x
+mkdir -p inst/faucet/daq-faucet-dot1x
 
-cp misc/faucet_dot1x.yaml inst/faucet/faucet_dot1x/faucet.yaml
+cp misc/faucet_dot1x.yaml.nodot inst/faucet/daq-faucet-dot1x/faucet.yaml
 
 sudo ovs-vsctl del-br br-dot1x || true
 
-docker rm -f faucet_dot1x || true
+docker rm -f daq-faucet-dot1x || true
```

Comment:
why not just daq-faucet?  Is there some expectation that this is running along side another faucet instance?

Reply:
Just to be safe. cmd/faucet dot1x <port> creates daq-faucet-dot1x. Same as why I opted for br-dot1x rather than br-faucet or br0

Reply:
Fair enough.